src/types.ts

```diff
@@ -7,6 +7,7 @@ export interface SipClientOptions {
   sipInfo: SipInfo;
   instanceId?: string; // ref: https://docs.oracle.com/cd/E95618_01/html/sbc_scz810_acliconfiguration/GUID-B2A15693-DA4A-4E24-86D4-58B19435F4DA.htm
   debug?: boolean;
+  clientId?: string;
```

Comment:
why do you need clientId? What is your use case? The SDK doesn't need a clientId to run.
And in your embeddable app, you can maintain the clientId. What's the point of adding clientId to this SDK (since it doesn't need it to run).

Reply:
Please refer to https://github.com/ringcentral/ringcentral-web-phone/commit/6d9d3762578efaeab4e713b99e915112b1a2953b
It provides documentation about how to add custom headers.